test/dialogs/dialog_api_request/properties_test.js

```diff
@@ -0,0 +1,58 @@
+require('../../setup');
+const GlobalHelpers = require('../../helpers');
+
+describe("DialogForm", function() {
+
+  const helpers = require("./helpers.js");
+  const c = helpers.constants;
+  const COMPONENT_ID = "form-dialog-properties-test";
```

Comment:
`const COMPONENT_ID = "form-dialog-properties-test";`

Think we've previously had clashes with tests, so they now use these IDs to reflect what is in the test 
(e.g. test for DialogApiRequest properties),

so...
`[const COMPONENT_ID = "dialogapirequest-properties-test";](url)`

or better still... assuming other similar IDs on elements (e.g. a container or something)
`[const COMPONENT_ID = "component-dialogapirequest-properties-test";](url)`